src/source/PeerConnection/JitterBuffer.c

```diff
@@ -344,13 +350,8 @@ STATUS jitterBufferFillFrameData(PJitterBuffer pJitterBuffer, PBYTE pFrame, UINT
     CHK(pJitterBuffer != NULL && pFrame != NULL && pFilledSize != NULL, STATUS_NULL_ARG);
     for (; UINT16_DEC(index) != endIndex; index++) {
         hashValue = 0;
-        retStatus = hashTableGet(pJitterBuffer->pPkgBufferHashTable, index, &hashValue);
+        CHK_STATUS(hashTableGet(pJitterBuffer->pPkgBufferHashTable, index, &hashValue));
```

Comment:
This is a behavior change in case the key is not present. Would be nice to comment why we the missing hash key is not expected.

Reply:
This is not a behavior change, it just changes which error code. If you returned STATUS_HASH_KEY_NOT_PRESENT then on line 354 you would return STATUS_NULL_ARG every time.